[2024-10-23T16:34:39.569+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-23T16:34:39.596+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DataPipeline.process_data manual__2024-10-23T16:34:28.412287+00:00 [queued]>
[2024-10-23T16:34:39.608+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DataPipeline.process_data manual__2024-10-23T16:34:28.412287+00:00 [queued]>
[2024-10-23T16:34:39.610+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-10-23T16:34:39.630+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_data> on 2024-10-23 16:34:28.412287+00:00
[2024-10-23T16:34:39.640+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=840) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-23T16:34:39.642+0000] {standard_task_runner.py:63} INFO - Started process 842 to run task
[2024-10-23T16:34:39.643+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DataPipeline', 'process_data', 'manual__2024-10-23T16:34:28.412287+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/src/***.py', '--cfg-path', '/tmp/tmpwexp3pxv']
[2024-10-23T16:34:39.646+0000] {standard_task_runner.py:91} INFO - Job 87: Subtask process_data
[2024-10-23T16:34:39.714+0000] {task_command.py:426} INFO - Running <TaskInstance: DataPipeline.process_data manual__2024-10-23T16:34:28.412287+00:00 [running]> on host 6c58daef65ee
[2024-10-23T16:34:39.888+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='MLopsProjectGroup6' AIRFLOW_CTX_DAG_ID='DataPipeline' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-23T16:34:28.412287+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-23T16:34:28.412287+00:00'
[2024-10-23T16:34:39.891+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-23T16:34:39.925+0000] {HandlingNullValues.py:24} INFO - Loading CSV data for processing
[2024-10-23T16:34:40.102+0000] {HandlingNullValues.py:40} INFO - No features dropped
[2024-10-23T16:34:40.790+0000] {HandlingNullValues.py:44} INFO - Processed data saved as CSV at /opt/***/dags/../data/processed/processed_data.csv
[2024-10-23T16:34:40.794+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-23T16:34:40.795+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-23T16:34:40.809+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DataPipeline, task_id=process_data, run_id=manual__2024-10-23T16:34:28.412287+00:00, execution_date=20241023T163428, start_date=20241023T163439, end_date=20241023T163440
[2024-10-23T16:34:40.862+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-23T16:34:40.885+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-23T16:34:40.888+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
